Makefile

```diff
@@ -40,6 +40,13 @@ pack:
 	cd tests/integration/charms/testdb && charmcraft pack
 	cd tests/integration/charms/testapp && charmcraft pack
 
+# Publish to PyPI
+.PHONY:
+publish:
+	rm -rf dist
+	uv build
+	uv publish
+
```

Comment:
Having this during the initial development phase seems reasonable, and I can definitely see an argument that allowing publishing to *test* PyPI with both trusted publishing and a token is reasonable.

However, the recommendation is that you don't allow token based publishing once you're all in on trusted publishing, so I don't think a `make publish` command is a good addition at this stage, unless it was specifically for publishing to *test* PyPI.